ext/native_test.go

```diff
@@ -211,6 +213,73 @@ func TestNativeTypesStaticErrors(t *testing.T) {
 	}
 }
 
+func TestNativeTypesJsonSerialization(t *testing.T) {
+	tests := []struct {
+		expr string
+		out  string
+	}{
+		{
+			expr: `[b'string']`,
+			out:  `["c3RyaW5n"]`,
+		},
+		{
+			expr: `TestAllTypes{
+				NestedVal: TestNestedType{
+					NestedListVal: ["first", "second"],
```

Comment:
Can we also add a test for maps? `NestedMapVal`

Reply:
Done.